cl/people_db/import_judges/populate_fjc_judges.py

```diff
@@ -641,3 +641,73 @@ def make_federal_judge(item, testing=False):
                 position.save()
             except Exception:
                 continue
+
+
+def make_mag_bk_judge(item, testing=False):
+    """Takes the federal judge data <item> and associates it with a Judge object.
+    Returns a Judge object.  To be used for importing bankruptcy and magistrate
+    judges.
+    """
+
+    if not testing:
+        name = "%s: %s %s" % (
+            item["CL_ID"],
+            item["NAME_FIRST"],
+            item["NAME_LAST"],
+        )
+        print("Now processing: %s" % name)
+
+        if not pd.isnull(item["NAME_MIDDLE"]):
+            if len(item["NAME_MIDDLE"]) == 1:
+                item["NAME_MIDDLE"] += "."
+
+        if not pd.isnull(item["GENDER"]):
+            gender = get_gender(item["GENDER"])
+
+        if not item["NAME_SUFFIX"] == '':
+            suffix = get_suffix(item["NAME_SUFFIX"])
+        else:
+            suffix = ''
+
+        # Instantiate Judge object.
+        person = Person(
+            name_first=item["NAME_FIRST"],
+            name_middle=item["NAME_MIDDLE"],
+            name_last=item["NAME_LAST"],
+            name_suffix=suffix,
+            gender=gender,
+            cl_id=item["CL_ID"],
+        )
+
+    if not testing:
+        person.save()
+
+    # Add position.
+    if re.search('Bankruptcy', item['COURT']):
+        position_type = "jud"
```

Comment:
`"jud"` and `"mag"` are both special secret values, right? We should grab these from the Position objects. It's going to be something like `Position.MAGISTRATE_JUDGE` or something, I think?

Reply:
Jon will update Positions table to include variables for these.